cmds/graph.c

```diff
@@ -796,7 +813,7 @@ static bool print_task_node(struct uftrace_task *task,
 
 	print_task_field(task);
 	pr_indent(indent_mask, indent, true);
-	pr_out("[%d] %s\n", task->tid, name);
+	pr_out("%s\n", name);
```

Comment:
I'm think of doing the same thing for function graph - moving call count to a field.  What do you think?  Obviously it's a separate change.

Reply:
I makes sense. I will do that later.